files/fluentd/run.sh

```diff
@@ -67,18 +66,36 @@ if [ $ALLOWED_PERCENT_OF_DISK -gt 100 ] || [ $ALLOWED_PERCENT_OF_DISK -le 0 ] ;
 fi
 # Determine allowed total given the number of outputs we have.
 ALLOWED_DF_LIMIT=$(expr $DF_LIMIT \* $ALLOWED_PERCENT_OF_DISK / 100) || :
+TOTAL_LIMIT_SIZE_PER_BUFFER=${TOTAL_LIMIT_SIZE_PER_BUFFER:-0}
+
+TOTAL_LIMIT_SIZE_PER_BUFFER=$(echo $TOTAL_LIMIT_SIZE_PER_BUFFER |  sed -e "s/[Kk]/*1024/g;s/[Mm]/*1024*1024/g;s/[Gg]/*1024*1024*1024/g;s/i//g" | bc) || :
+
+# total limit size requested per buffer
+TOTAL_LIMIT_SIZE_PER_BUFFER=$(expr $TOTAL_LIMIT_SIZE_PER_BUFFER / $NUM_OUTPUTS) || :
```

Comment:
Each output can have a maximum of 1 buffer which means there is no need to divide here since  we are specifiying the max of the buffer via an env var.

Reply:
@ajaygupta978  these was not addressed

Reply:
@jcantrill I was waiting for your comments for go code. I already tested run.sh with your suggested changes. I will push go and run.sh changes together.

Reply:
Done.